firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Array.txt

```diff
@@ -0,0 +1,25 @@
+SCENARIO("Create an array", "[array]")
+  GIVEN("A set of int values '\x0f\x36\x08\xff\x89\x01\x80\x02' ")
+  WHEN("An array of given int size is created properly")
+     THEN("Value at index 5 in array is as expected")
```

Comment:
Since we have knowledge about what the value at every index in the array should be, let's test every index, e.g. in a for loop comparing the returned array with a pre-constructed array containing the expected values in the expected order.